pkg/controller/image-stream-tag-reconciler/reconciler.go

```diff
@@ -162,18 +189,20 @@ func (r *reconciler) reconcile(req controllerruntime.Request, log *logrus.Entry)
 	return nil
 }
 
-func (r *reconciler) isPublishedFromCIOperatorRepo(log *logrus.Entry, ist *imagev1.ImageStreamTag) (*cioperatorapi.ReleaseBuildConfiguration, bool) {
```

Comment:
(can't comment on the right line, just adding here)

You wrote:

```go
if istHasBuildRunning || err != nil {
	return wrapErrIfNonNil("failed to check if there is a running build for imageStreamTag", err)
}
```

You enter the `if` with two disjoint cases but then exit with two disjoint actions ... why is that better than

```go
if err != nil {
    return fmt.Errorf("failed to check if there is a running build for imageStreamTag", err)
}
if istHasBuildRunning {
    return nil
}
```